akka-samples/akka-docs-java-lambda/pom.xml

```diff
@@ -6,7 +6,7 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <publishedAkkaVersion>2.3.7</publishedAkkaVersion>
+    <publishedAkkaVersion>2.3.9</publishedAkkaVersion>
```

Comment:
Is this how it is done in the other samples, or do they use `2.3-SNAPSHOT` in order to pick up the automatic rename?